versioned/storage/dynamodb/src/main/java/org/projectnessie/versioned/storage/dynamodb/DynamoDBPersist.java

```diff
@@ -355,7 +355,7 @@ private void findReferencesPage(
   @jakarta.annotation.Nonnull
   public Obj fetchObj(@Nonnull @jakarta.annotation.Nonnull ObjId id) throws ObjNotFoundException {
     GetItemResponse item =
-        backend.client().getItem(b -> b.tableName(TABLE_OBJS).key(objKeyMap(id)));
+        backend.client().getItem(b -> b.tableName(this.backend.tableObjs).key(objKeyMap(id)));
```

Comment:
Would you mind dropping `this` qualified for `backend`? It was previously referenced by unqualified names in other places (except the constructor, of course). Even the first usage on this line in unqualified.